index.html

```diff
@@ -0,0 +1,364 @@
+                  Change the colors, pictures or<br />any of the sections to
+                  suit your<br />needs.
+                </p>
+              </div>
+            </div>
+
+            <div class="advantages-box">
+              <div class="advantages-box__icon">
+                <img
+                  class="advantages-box__icon-svg"
+                  src="./images/design.svg"
+                  alt="Pennant icon"
+                />
+              </div>
+
+              <h2 class="advantages-box__title">
+                SLICK&nbsp;AND&nbsp;BEAUTIFUL<br />DESIGN
+              </h2>
+              <div class="advantages-box__benefits">
+                <p class="advantages-box__benefits-description">
+                  Trendy and fresh design, fits<br />any website.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="section-3__background">
+        <div class="section section-3">
+          <div class="recommendation">
+            <img
+              class="recommendation__image"
+              src="./images/logos.png"
+              alt="Logos of interesting websites"
+            />
+          </div>
+        </div>
+      </section>
+
+      <section class="section-4__background">
+        <div class="section-4">
+          <div class="imac-monitor">
+            <img
+              class="imac-monitor__image"
+              src="./images/imac.png"
+              alt="Apple monitor logo"
+            />
+          </div>
+
+          <div class="nova-theme">
+            <h1 class="nova-theme__title">SIMPLE&nbsp;AND&nbsp;BEAUTIFUL</h1>
+            <p class="nova-theme_description">
+              Use Nova theme for your next web project. It is<br />completely
+              customizable so you can change any of<br />the sections to fit
+              your needs. Nova Theme is Free for<br />any kind of use, presonal
+              and commercial. Have fun<br />and good luck!
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="section-5__background">
+        <div class="section section-5">
+          <div class="prices-header">
+            <h1 class="prices-header__title">
+              CHOOSE&nbsp;YOUR&nbsp;PRICING&nbsp;PLAN
+            </h1>
+
+            <p class="prices-header__subtitle">
+              Pick any of our super affordable pricing plans
+            </p>
+          </div>
+
+          <div class="price-offers">
+            <div class="price-offer">
+              <h2 class="price-offer__variant">BASIC</h2>
+              <div class="price-offer__price">
+                <p class="price-offer__price-value">$15</p>
+                <p class="price-offer__price-months-period">per month</p>
+              </div>
+
+              <p class="price-offer__benefits">
+                Up to 7 Projects<br />
+                2 Additional Developers
+              </p>

```

Comment:
Tutaj zdecydowanie lista bo to lista elementów zawierających się w pakiecie.

PS. Tak, listy można w sobie zagnieżdżać :)